.github/workflows/cmake.yml

```diff
@@ -71,11 +75,19 @@ jobs:
           vcpkgDirectory: ${{ runner.workspace }}/vcpkg
           vcpkgGitCommitId: 78b61582c9e093fda56a01ebb654be15a0033897 # HEAD on 2023-08-6
 
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1.2
+        with:
+          variant: sccache
```

Comment:
I'd suggest using sccache's built-in Actions cache support. allwpilib is almost definitely large enough that a local disk cache will end up evicting all the cache.

Reply:
The cache size based on initial runs is only 100 MB, but I'll look into the built-in actions support.

Reply:
Yeah, I think the default max cache size from this action is 100 MB. So the cache is full and being LRU evicted.

Reply:
No, the default is 500 MB.

Reply:
Thanks for the recommendation on sccache, it actually seems to be working and getting cache hits.